src/main/java/org/jitsi/jigasi/GatewaySession.java

```diff
@@ -44,6 +45,19 @@
     private final static Logger logger = Logger.getLogger(GatewaySession.class);
 
     /**
+     * Property name of extra INVITE header which specifies password required
+     * to enter MUC room that is hosting the Jitsi Meet conference.
+     */
+    public String JITSI_MEET_ROOM_PASS_HEADER = "Jitsi-Conference-Room-Pass";
```

Comment:
Why is field named like constant ? It seems to also serve the role of a default value, but then it should be a static field DEFAULT_JITSI_MEET_ROOM_PASS_HEADER.

Please separate into field and static default value.